dbms/src/Storages/StorageDeltaMerge.h

```diff
@@ -238,6 +238,11 @@ class StorageDeltaMerge
 
     mutable std::mutex decode_schema_mutex;
     DecodingStorageSchemaSnapshotPtr decoding_schema_snapshot;
+    // The following two members must be used under the protection of table structure lock
+    bool decoding_schema_changed = false;
+    // internal version for `decoding_schema_snapshot`
+    size_t decoding_schema_version = 1;
```

Comment:
We use `Int64` as the param type in `void releaseDecodingBlock(Int64 block_decoding_schema_version, ...`, should also use Int64 here?

Reply:
Fixed.